go/checks/checks.go

```diff
@@ -178,4 +178,12 @@ func CheckMaxPartitionsContributed(label string, maxPartitionsContributed int64)
 		return fmt.Errorf("%s: MaxPartitionsContributed is %d, should not be negative", label, maxPartitionsContributed)
 	}
 	return nil
-}
\ No newline at end of file
+}
+
+// CheckConfidenceLevel returns an error if the supplied confidence level is not between 0 and 1
```

Comment:
`// CheckConfidenceLevel returns an error if the supplied confidence level is not between 0 and 1 (exclusive).`

Reply:
Sorry, it seems it is inclusive.